README.md

```diff
@@ -1,46 +1,72 @@
+# Formula 1 2022
 
-## Available Scripts
+> SPA containing information about 2022 season, where you can find past season pilots, statistics and Circuits.
 
-In the project directory, you can run:
+![coverPage](https://user-images.githubusercontent.com/99816838/194368198-e0d2d003-1bed-43b7-8bd9-a389afbe18f2.png)
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- React/Redux
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Live Version
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+[Formula 1](https://preeminent-sable-64e23f.netlify.app)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Loom Video
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Loom Video](https://www.loom.com/share/516443cc5529418d975f8aa602293636)
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Getting Started
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+To get a local copy up and running follow these simple example steps.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Clone this repository: https://github.com/Flunavgt/formula1_capstone.git
 
-## Learn More
+Install dependencies with: "npm i"
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+run on local server with: "npm start"
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Ackowledgments
+
+  * To [Nelson Sakwa](https://www.behance.net/sakwadesignstudio), Original design.
+
+
+
+
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.